code/controllers/CMSPageHistoryController.php

```diff
@@ -358,7 +358,7 @@ public function ShowVersionForm($versionID = null) {
 	/**
 	 * @param int $versionID
 	 * @param int $otherVersionID
-	 * @return Form
+	 * @return mixed Form|boolean|SS_HTTPResponse
```

Comment:
@dhensby Explicitly adding a pipe-delimited list of the available return types _is permitted_ in phpDoc,:
 
>If you want to explicitly show multiple possible return types, list them pipe-delimited without spaces (e.g. "@return int|string").

https://manual.phpdoc.org/HTMLSmartyConverter/HandS/phpDocumentor/tutorial_tags.return.pkg.html